xml/depl_require.xml

```diff
@@ -2069,6 +2069,24 @@
         between the cluster nodes) to ensure redundancy.
        </para>
       </important>
+      <important>
+       <title>Dedicated networks</title>
+       <para>
+        The <literal>corosync</literal> network communication layer is
```

Comment:
My proposal (sentence length, clarity, plus trying to integrate feedback by @roseswe on https://github.com/SUSE-Cloud/doc-cloud/pull/30):
[...] critical to the health of a cluster. Therefore, use a dedicated bond for both Admin network and corosync communication and a second bond for the OpenStack traffic.